src/main/groovy/edu/oregonstate/mist/locations/core/DayOpenHours.groovy

```diff
@@ -4,10 +4,10 @@ import com.fasterxml.jackson.annotation.JsonFormat
 import com.fasterxml.jackson.annotation.JsonIgnore
 
 class DayOpenHours {
-    @JsonFormat(shape=JsonFormat.Shape.STRING, pattern="HH:mm Z", timezone="UTC")
+    @JsonFormat(shape=JsonFormat.Shape.STRING, pattern="yyyy-MM-dd HH:mm a Z", timezone="UTC")
     Date start
 
-    @JsonFormat(shape=JsonFormat.Shape.STRING, pattern="HH:mm Z", timezone="UTC")
+    @JsonFormat(shape=JsonFormat.Shape.STRING, pattern="yyyy-MM-dd HH:mm a Z", timezone="UTC")
```

Comment:
The "a" in the pattern is not needed. It adds "AM/PM", but we are using the 24hr format. 

We are trying to use: ISO 8601 for our dates in APIs: https://en.wikipedia.org/wiki/ISO_8601 . Check and see if there's a constant already defined by Java that we can use to specify this well known date format.